modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -51,20 +49,23 @@ You can create more than one API specification in a single project. If you want
 As you draft, the text editor suggests RAML nodes, methods, and other elements that you can add at the location of your cursor.
 +
 You can import files into your project, either separately or bundled together in .zip files. The files can be RAML 1.0 files, JSON files, or OpenAPI Specification (OAS) 2.0 files. The files can be on your computer or you can specify a URL for them if they are located online.
-+
-//For more information about importing files and steps for how to import them, see "Import Files into an API Project", which is linked to from the *See also* section at the end of this topic.
-
 
 
 == What to Do Next
 
-You can publish the API fragment to Anypoint Exchange. Click the *Publish* button that is in the top-right corner of the text editor, and then click *Publish to Exchange* to open the *Publish API specification to Exchange* dialog.
+You can publish the API fragment to Anypoint Exchange. Click the *Publish* button that is in the top-right corner of the text editor, and then click *Publish to Exchange*. Specify a version number for the project, which Exchange treats as an _asset_. This number is displayed in Exchange in the *Version* section of the right-hand pane in the page for the project.
```

Comment:
@iamerico , the whole sentence in the doc is "Specify a version number for the project, which Exchange treats as an _asset_." So, I do use the term _asset_. However, I use the word _project_ because the entire content of the project is published, not just the root file. If I went with your suggestion, the sentence would imply that only the API fragment that is set as the root file is published.

Reply:
Ok, that makes sense.